tests/test_api/test_models/test_student/test_student_serializer.py

```diff
@@ -0,0 +1,52 @@
+import json
+import unittest
+
+from api.models.enums import Relationship
+from api.models.student.student import Student
+from api.models.student.student_serializer import StudentSerializer
+from api.models.team import Team
+
+
+class TestStudentSerializer(unittest.TestCase):
+    @classmethod
+    def setUp(cls):
+        cls.students = [
+            Student(
+                _id=1,
+                name="Harry",
+                attributes={4: [2, 56, 1], 11: [2], 234: [-1, 0]},
+                relationships={2: Relationship.FRIEND},
+                project_preferences=[6, 1],
+            ),
+            Student(
+                _id=2,
+                name="Jane",
+                attributes={4: [31, 2], 11: [], 234: [1, -1]},
+                relationships={1: Relationship.FRIEND},
+                project_preferences=[3, 2],
+            ),
+        ]
+        cls.json_students = [
+            '{"_id": 1, "name": "Harry", "attributes": {"4": [2, 56, 1], "11": [2], "234": [-1, 0]}, "relationships": {"2": -1}, "project_preferences": [6, 1]}',
+            '{"_id": 2, "name": "Jane", "attributes": {"4": [31, 2], "11": [], "234": [1, -1]}, "relationships": {"1": -1}, "project_preferences": [3, 2]}',
+        ]
+
+    def test_student_serializer__encode_returns_string(self):
```

Comment:
I don't know if we need this method. It does not actually test what encode returns because encode returns a dict. It checks that the returned dict gets turned into a string. Let me know what you think.

Reply:
Yeah I'd say not super needed here tbh